pkg/appgw/backendhttpsettings.go

```diff
@@ -18,6 +18,11 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	// DefaultConnDrainTimeoutInSec provides default value for ConnectionDrainTimeout
+	DefaultConnDrainTimeoutInSec = 30
```

Comment:
I wonder if we can get these consts from App Gateway on runtime so we don't replicate the values in this repo (just a thought)

Reply:
no we don't have any such api as of now :/